tests/integration_tests/functional/test_balloon.py

```diff
@@ -22,10 +22,22 @@
 MB_TO_PAGES = 256
 
 
-def get_rss_mem_by_pid(pid):
+def get_stable_rss_mem_by_pid(pid):
     """Get the RSS memory that a guest uses, given the pid of the guest."""
-    output = subprocess.check_output("pmap -X {}".format(pid), shell=True)
-    return int(output.decode('utf-8').split('\n')[-2].split()[1], 10)
+
+    def get_rss_from_pmap():
+        output = subprocess.check_output("pmap -X {}".format(pid), shell=True)
```

Comment:
For logging purposes, you can use `utils.run_cmd()`.

It's also helpful in avoiding `decode` calls.

Reply:
Updated it